src/ui/Logic/UiUtil.cs

```diff
@@ -530,6 +524,18 @@ public static void InitializeSubtitleFont(Control control)
             }
         }
 
+        private static void SetControlSubtitleFont(Control control, string subtitleFontName, int subtitleFontSize, bool subtitleFontBold)
+        {
+            if (subtitleFontBold)
```

Comment:
You can single line this if you want.

`control.Font = new Font(subtitleFontName, subtitleFontSize,  subtitleFontBold ? FontStyle.Bold : FontStyle.Regular)`